src/cluster/src/cli/start/sys.rs

```diff
@@ -0,0 +1,40 @@
+use crate::cli::start::StartOpt;
+use crate::cli::ClusterCliError;
+use crate::sys::{SysConfig, SysInstaller};
+use crate::ClusterError;
+use crate::error::SysInstallError;
+
+pub fn install_sys(opt: StartOpt, upgrade: bool) -> Result<(), ClusterCliError> {
+    install_sys_impl(opt, upgrade).map_err(ClusterError::InstallSys)?;
+    Ok(())
+}
+
+fn install_sys_impl(opt: StartOpt, upgrade: bool) -> Result<(), SysInstallError> {
```

Comment:
why creating another fn?

Reply:
Maybe whole thing should be moved into `InstallSys`

Reply:
This was done just to convert all of the error types at once. Otherwise, we would need map_err all over the place which is messy